doc/source/user/absolute_beginners.rst

```diff
@@ -280,7 +280,7 @@ and order when you call the function. `Read more about sorting an array here <ht
 
 If you start with this array::
 
-  >>> arr = np.array([2, 1, 5, 3, 7, 4, 6, 8])
```

Comment:
I think this prommpt is needed for refguide to work, although @mattip will know more

Reply:
Thank you! Happy to adjust as needed

Reply:
I added back the prompts, but let me know if you want that changed.